docs/sources/architecture/thing-definition.md

```diff
@@ -154,4 +154,49 @@ The channel group type is defined on the same level as the thing types and chann
 </channel-group-type> 
 ```
 
-When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
\ No newline at end of file
+When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
+
+## Properties
+The configuration descriptions explained in full detail [here](../architecture/configuration.md) are used in order to provide a flexible configuration mechanism for bindings. However as a binding developer you will often have the demand to add meta data to your thing type that represent no configuration option. For this purpose the thing type definition provides the possibility to specify properties, e.g. the vendor or the model ID of the concrete thing:
+
+```xml 
+    <thing-type id="thingTypeID">
+        ...
+        <properties>
+             <property name="vendor">MyThingVendor</property>
+             <property name="modelId">...</property>
+        </properties>
+		...
+    </thing-type>
+```
+
+These properties can be retrieved as a key value map by the operation `getProperties` of the `org.eclipse.smarthome.core.thing.Thing` interface. Additional thing properties that are only known at runtime have to be set by the concrete thing handler. For this purpose the operation `setProperty` of the thing interface has to be used:
```

Comment:
Please make clear here, that the properties which are specified in XML are part of the ThingType. If a thing will be created for this ThingType, the properties will be copied to the thing, so that they are available also there.